app/lib/specialist_publisher.rb

```diff
@@ -37,6 +38,8 @@ def document_types
     "vehicle_recalls_and_faults_alert" => VehicleRecallsAndFaultsAlertObserversRegistry,
   }.freeze
 
+  private
```

Comment:
I'll push a fix putting private keyword back in original position, and removing the duplicated mapping.